### PR TITLE
[PowerPoint] (PowerPointApi 1.5) Releasing PowerPointApi 1.5 to GA

### PR DESCRIPTION
**Pull request by alison-mk**
Updates to the following articles will be added in a later PR. Can't update these articles right now because I don't have platform availability information. Working on tracking it down. 
- /docs/requirement-sets/requirement-sets.md
- /docs/requirement-sets/powerpoint/powerpoint-api-requirement-sets.md

### CODE REVIEW

File: docs/requirement-sets/powerpoint/powerpoint-api-1-5-requirement-set.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: PowerPoint JavaScript API requirement set 1.5
+description: Details about the PowerPointApi 1.5 requirement set.
+ms.date: 11/11/2022
+ms.prod: powerpoint
+ms.localizationpriority: medium
+---
+
+# What's new in PowerPoint JavaScript API 1.5
+
+PowerPointApi 1.5 added APIs to select slides, text ranges, and shapes within presentations. Prior to this release, selecting slides and shapes required a complicated series of steps. These selection APIs enable developers to directly select slides, text ranges, and shapes, creating a more efficient and intuitive add-in development process.
```
Comment (alison-mk) [inline docs/requirement-sets/powerpoint/powerpoint-api-1-5-requirement-set.md:11]: @Rick-Kirkham Can you take a look at this article in particular? Want to make sure I covered the main benefits.

**Comment by alison-mk** [comment]
Requesting a review from @ElizabethSamuel-MSFT as a docs generation process expert, and requesting a review from @Rick-Kirkham as a subject matter (PowerPointApi 1.5) expert. 

Note that updates to the following two articles in this repo must be added in a later PR. I can't update these articles right now because I don't have platform availability information — still working on tracking it down. Not sure how long getting that info will take, and I don't want to hold up merging this PR. 
- /docs/requirement-sets/requirement-sets.md
- /docs/requirement-sets/powerpoint/powerpoint-api-requirement-sets.md

File: .openpublishing.publish.config.json [hunk 2]
```diff
@@ -263,6 +263,10 @@
       "ReferenceTOC": "docs/docs-ref-autogen/powerpoint_1_4/toc.yml",
       "ConceptualTOCUrl": "/office/dev/add-ins/toc.json"
     },
+    {
+      "ReferenceTOC": "docs/docs-ref-autogen/powerpoint_1_5/toc.yml",
```
Comment (ElizabethSamuel-MSFT) [inline .openpublishing.publish.config.json:267]: I'm not seeing powerpoint-js-1.5 in the moniker_ranges array

File: docs/requirement-sets/toc.yml [hunk 3]
```diff
@@ -241,6 +241,9 @@
     - name: PowerPointApi preview requirement set
       href: ../../requirement-sets/powerpoint/powerpoint-preview-apis.md
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/toc.yml:242]: Make sure to update the preview APIs file too

File: .openpublishing.publish.config.json (hunk 2 above)
Comment (alison-mk) [reply .openpublishing.publish.config.json:268]: Thanks for the review! I fixed both issues you caught.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**